oauth2_http/javatests/com/google/auth/oauth2/UserAuthorizerTest.java

```diff
@@ -148,6 +149,7 @@ public void getAuthorizationUrl() throws IOException {
             .setScopes(DUMMY_SCOPES)
             .setCallbackUri(CALLBACK_URI)
             .setUserAuthUri(AUTH_URI)
+            .setPKCEProvider(pkce)
```

Comment:
Can we add a test case of without calling `setPKCEProvider()`, which should be the majority of existing customers' usage? We might not be able to check the code_challenge value, but can validate it empty or not.

Reply:
The rest of the test cases are using PKCE because the builder will build the default PKCE provider if one isn't injected.

This test case is different because I want to verify against a well known PKCE object.